downstream/modules/platform/ref-controller-policy-considerations.adoc

```diff
@@ -0,0 +1,11 @@
+[id="controller-policy-considerations"]
+
+= Notable policy considerations
+
+Take the following policy considerations into account:
+
+* Both `policy_instance_percentage` and `policy_instance_minimum` set minimum allocations. 
+The rule that results in more instances assigned to the group takes effect. 
+For example, if you have a `policy_instance_percentage` of 50% and a policy_instance_minimum of 2 and you start 6 instances, 3 of them would be assigned to the Instance Group. If you reduce the number of total instances in the cluster to 2, then both of them are assigned to the instance group to satisfy policy_instance_minimum. 
```

Comment:
Should both instances of _policy_instance_minimum_ in this sentence be surrounded in single quotes?